deploy/kubectl_helpers/utils/hail-search.yaml

```diff
@@ -0,0 +1,66 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: hail-search
+  namespace: seqr-prototype
+  labels:
+    name: hail-search
+    deployment: prototype
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 5000
+  selector:
+    name: hail-search
+---
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name:  hail-search
+  namespace: seqr-prototype
+  labels:
+    app.kubernetes.io/name:  hail-search
+    name:  hail-search
+    deployment: prototype
+spec:
+  replicas: 4
+  selector:
+    matchLabels:
+      name: hail-search
+  template:
+    metadata:
+      labels:
+        name: hail-search
+        app.kubernetes.io/name: hail-search
+        deployment: prototype
+    spec:
+      volumes:
+      - name: hail-datasets
+        gcePersistentDisk:
```

Comment:
I think this `gcePersistentDisk` volume driver is actually deprecated -- you'll want to use a persistentVolume and claim: https://cloud.google.com/kubernetes-engine/docs/how-to/persistent-volumes/readonlymany-disks#pre-existing-persistent-disk